### PR TITLE
Split docker-compose into regular and build

### PR DESCRIPTION
**Pull request by Demonsthere**
**Description**
- Remove building from main compose file
- Create second that only builds
- Remove unused bitbucket pipeline file

### CODE REVIEW

File: docker-compose-build.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+version: '3'
+services:
+  orcpub:
+    build:
```
Comment (MeanderingCode) [inline docker-compose-build.yaml:5]: Please add `image: orcpub:latest` for better ease of development to allow only rebuilding when desired via a `docker-compose up --build`

File: docker-compose-build.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      EMAIL_SECRET_KEY: ''
+      EMAIL_SERVER_PORT: 587
+      # Email address to send errors to
+      EMAIL_ERRORS_TO: ''
+      DATOMIC_URL: datomic:free://datomic:4334/orcpub
+    depends_on:
+      - datomic
+    restart: always
+  datomic:
+    build:
```
Comment (MeanderingCode) [inline docker-compose-build.yaml:21]: Same here, with `image: datomic-free:latest`

**Review by MeanderingCode — DISMISSED** [review]
This is great, thank you.

Request to add non-prefixed image tags for local use.

File: docker-compose-build.yaml (hunk 1 above)
Comment (Demonsthere) [reply docker-compose-build.yaml:5]: I assumed we want to have the second compose for dedicated bulding, and the first for dedicated image usage. Of course I can add those, but that will require the `--build` clause. I assumed we want to have the default behaviour to be build on `docker-compose up`

**Comment by Demonsthere** [comment]
I would like to merge this as discussed on discord, the images are named as the docker services (orcpub and datomic), so no changes are needed.

**Review by codeGlaze — APPROVED** [review]

**Pull request merged**